test/cpp/microbenchmarks/fullstack_fixtures.h

```diff
@@ -41,6 +41,7 @@ extern "C" {
 #include "test/core/util/port.h"
 }
 
+#include "src/cpp/client/create_channel_internal.h"
```

Comment:
If you're adding it here, please remove it from the other places where it is used (each of the fullstack_*.h files that includes this fixtures header)

Reply:
Done